druid/website/druid.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kubedb.com/v1alpha2
+kind: Druid
+metadata:
+  name: druid
+  namespace: demo
+spec:
+  version: 28.0.1
+  deepStorage:
+    type: s3
+    configSecret:
+      name: deep-storage-config
+  metadataStorage:
+    name: mysql-demo
+    namespace: demo
+    createTables: true
+  zookeeperRef:
+    name: zk-demo
+    namespace: demo
+  topology:
+    coordinators:
+      replicas: 1
+    brokers:
+      replicas: 1
+    historicals:
+      replicas: 1
+      storage:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+        storageClassName: standard
```

Comment:
Remove the `storageClassName` & `accessModes` parts. 
Check with applying on the cluster